mmv1/templates/terraform/constants/cloud_run_service.go.erb

```diff
@@ -19,6 +19,23 @@ func cloudrunAnnotationDiffSuppress(k, old, new string, d *schema.ResourceData)
 		return true
 	}
 
+	// For other keys, don't suppress diff.
+	return false
+}
+
+var cloudRunGoogleProvidedLabels = regexp.MustCompile(`cloud\\.googleapis\\.com/(?:(?:location))`)
```

Comment:
This doesn't seem to work for me unless the regex string is: 
```suggestion
var cloudRunGoogleProvidedLabels = regexp.MustCompile(`cloud\.googleapis\.com/(?:(?:location))`)
```
Which leads me to discover that the Annotations diff suppress isn't working correctly either. Looking back, this change seems to start it?:
https://github.com/GoogleCloudPlatform/magic-modules/commit/0b23f72f9124431038e423da5e2a4e6230883a54

The use of backticks instead of quotes seemed to change the number of backlashes needed to escape characters. Are you experiencing this as well?

Reply:
Hmm. I think it works. I checked the regex online to validate it

Reply:
The actual regex is fine, I think it is behavior in regexp. Using backticks vs quotes changes the escape behavior: https://stackoverflow.com/a/50268933 
I saw unsuppressed diffs when I built this and applied
```
resource "google_cloud_run_service" "default" {
  name     = "helloworld"
  location = "us-central1"

  metadata {
    labels = {
      label1 = "key1"
    }
    annotations = {
      label2 = "key2"
    }
  }
  template {
    spec {
      containers {
        image = "gcr.io/cloudrun/hello"
        env {
          name  = "SOURCE"
          value = "remote"
        }
      }
    }
  }
  autogenerate_revision_name = true
}
```

Reply:
I committed your recommendation. Try it again

Reply:
confirmed, thanks